test/modules/ModuleRestarter/src/Program.cs

```diff
@@ -38,39 +32,51 @@ static async Task<int> MainAsync()
         }
 
         /// <summary>
-        /// Restarts random modules periodically (with default restart occurrence once every 10 minutes)
+        /// Restarts random modules periodically (with default restart occurrence once every 10 minutes).
         /// </summary>
         static async Task RestartModules(CancellationTokenSource cts)
         {
+            if (Settings.Current.DesiredModulesToRestart.Count == 0)
+            {
+                Logger.LogInformation("No modules names found in input. Stopping.");
+                return;
+            }
+
```

Comment:
nit: I feel like it is better to raise this method to make the behavior obvious from MainAsync().